scripts/js/runtime-upgrade-remote/package-lock.json

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
I was wondering if we needed to commit this, and yes, we need it: https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
I would prefer the `yarn.lock` to follow the `runtime-upgrade-local` path. AFAIK, yarn locks are preferred.

Reply:
done in 4e38fda8b5550d955907c227ca518476171054bc